js/flightlog_fields_presenter.js

```diff
@@ -215,7 +215,28 @@ function FlightLogFieldPresenter() {
 							'debug[1]':'debug[1]',
 							'debug[2]':'debug[2]',
 							'debug[3]':'debug[3]',    		
-    		}
+    		},
+    		'DEBUG_FFT' : {
+    		                'debug[all]':'Debug FFT',
+    		                'debug[0]':'gyro_preFFT[roll]',
```

Comment:
FFT (fast fourier transform) is the process used to calculate the frequency spectrum. But what is actually debugged here is the dynamic notch filter. (Probably that should be renamed...)
I'd prefer these names:
0: gyro_raw[roll]
1: gyro_dyn_notch[roll]
2: gyro_bpf[roll]
3: fft center index[roll]